lexicon/conversion.py

```diff
@@ -140,6 +140,118 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from space or tab separated files.
+
+    As the splitting is taken from RASR and not fully tested,
+    it might not work in all cases so do not use this job
+    without checking the output manually on new lexicas.
+    """
+
+    def __init__(self, text_file, add_unknown, add_noise, compressed=True):
+        """
+
+        :param Path text_file:
+        :param bool add_unknown: add [UNKNOWN] lemma
+        :param bool add_noise: add [NOISE] lemma
+        :param compressed: save as .xml.gz
+        """
+        self.text_file = text_file
+        self.add_unknown = add_unknown
+        self.add_noise = add_noise
+
+        self.out_bliss_lexicon = self.output_path(
+            "lexicon.xml.gz" if compressed else "lexicon.xml"
+        )
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        lex = lexicon.Lexicon()
+
+        lex.add_lemma(
+            lexicon.Lemma(
+                orth=["[SILENCE]"],
```

Comment:
special lemma "silence" should have an empty orth tag. Otherwise no silence will be aligned between words if not explicitly transcribed.
```suggestion
                orth=["[SILENCE]", ""],
```